kindtest.sh

```diff
@@ -237,9 +242,19 @@ rm -rf "${RESULT_ROOT:?}/*"
 
 echo 'Run tests...'
 if [ "${test_filter}" = "ItOperatorUpgrade" ] || [ "${parallel_run}" = "false" ]; then
-  echo "Running mvn -Dit.test=${test_filter} -Dwdt.download.url=${wdt_download_url} -Dwit.download.url=${wit_download_url} -pl integration-tests -P integration-tests verify"
-  time mvn -Dit.test="${test_filter}" -Dwdt.download.url="${wdt_download_url}" -Dwit.download.url="${wit_download_url}" -pl integration-tests -P integration-tests verify 2>&1 | tee "${RESULT_ROOT}/kindtest.log"
+  if [ "${verify_image_run}" != "verify-images-tests" ]; then
```

Comment:
you don't need this condition, use ${maven_profile_name} in the mvn command

Reply:
true. Thanks

Reply:
fixed